prime-router/src/main/kotlin/azure/SubmissionsFacade.kt

```diff
@@ -44,9 +44,10 @@ class SubmissionsFacade(
         sortOrder: String,
         sortColumn: String,
         offset: OffsetDateTime?,
+        toEnd: OffsetDateTime?,
         pageSize: Int
     ): String {
-        val result = findSubmissions(organizationName, sortOrder, sortColumn, offset, pageSize)
+        val result = findSubmissions(organizationName, sortOrder, sortColumn, offset, toEnd, pageSize)
```

Comment:
This may not be a problem because Kotlin does the right thing, but it bothers my mental typechecker, so I'm going to do a quick rant.

`<rant>`
I wish we used named arguments more in our Kotlin code. When adding optional parameters in the middle of list of function arguments the same type (resultsAfterDate and resultsBeforeDate), it's hard for the typechecker to figure out what's missing.

I think this says both `OffsetDateTime?` and `OffsetDateTime?` are optional?

If that's true and just one of the values for `resultsAfterDate` or `resultsBeforeDate` is missing, which one should be matched? (In other words, is it possible to leave off `offset` and just supple `toEnd`?) Naturally, this doesn't really make sense, but I'm not sure how the Kotlin typechecker figures this out.

If we standardized on named arguments, then the call would be more like
`findSubmissions(organization=organizationName, qSortOrder=sortOrder, qSortColumn=sortColumn, resultsAfterDate=offset, resultsBeforeDate=toEnd, pageSize=pageSize)`, then when we leave off a parameter, the typechecker KNOWS which one is missing explicitly. 
`</rant>`

Optional function arguments in the middle of the argument list may not be an issue in Kotlin, but it would be for many other languages.

Reply:
It's not saying the parameter is optional, it's saying the value is nullable. In Kotlin, `date: OffsetDateTime?` is how you would type the equivilent of `date: DateTime | null` in TypeScript. The params are all required, it's just possible that they'll be null and those null checks are useful in deeper functions.

Reply:
Oh... lol... so much for my Kotlin knowledge. 🤦  Thanks. (Glad I made this a comment and not an issue)